### ISSUE TITLE
[BUG_SP] Tractor with Seeder or Plow continues reversing when lining up to new path

### ISSUE THREAD

**Issue by BippyMiester** [Courseplay/Courseplay_FS22#2830]
**Steps to reproduce**
Generate a course on a irregular field, that has a u-shaped bend with a seeder, or a plow.

**Actual behavior**
The reason I suspect that the issue is the same is due to the irregular field.

My issue is with a plow, and a seeder. Currently whenever the tractor reverses to line up with the next pass, it just keeps reversing. It doesn't stop until it hits a tree. Then it says "Blocked by object". I've taken a screenshot, hopefully this maybe helps a little bit.

![image](https://github.com/Courseplay/Courseplay_FS22/assets/4575704/6ebb07e0-e1d3-4075-8c77-3acb9cf2103b)

Also here is a screenshot of the field, and the path, as well as my vehicle settings and CP settings to hopefully help in the debugging process.

![image](https://github.com/Courseplay/Courseplay_FS22/assets/4575704/d82e64a5-519a-49f4-88e9-b41b15994499)
![image](https://github.com/Courseplay/Courseplay_FS22/assets/4575704/25dbb604-b2f5-4d10-aef6-4d671b6d66ce)
![image](https://github.com/Courseplay/Courseplay_FS22/assets/4575704/58128cb8-4eb9-412f-a2c8-53f7884eca93)
![image](https://github.com/Courseplay/Courseplay_FS22/assets/4575704/3e847904-69fe-4124-8cae-315449e0d32d)

Will attach a video of the entire source being run with timestamps of the issue being happening. I just regenerated the route, using the same settings, and I'm just waiting for the issue to happen again.

**Courseplay version**
Version used: 7.3.1.2

**Log**
Will post log once the video is done.

**Map**
The Western Wilds

**Vehicles/ implements involved**
Which vehicle or implement?
DEUTZ-FAHR 8280 TTV
Mod yes/no ?
No

**Relevant script mods**
Autodrive is installed but not used in this course

**Comment by Tensuko** [comment]
Your screenshot shows a modded"plow" wich are mostly bugged.
And there is no logfile for your seeder problem.
With that plow -> wont fix
If the same happens with a seeder, need log with debug active.

**Comment by BippyMiester** [comment]
> Your screenshot shows a modded "plow" wich are mostly bugged.

Could be the plow, yes. Will try with a different plow and report back.

Current plow, I forgot that it was a mod to be honest. Its the Rostalmash K-12200 plow by Nels0n. Link to download is below.

https://fs22.com/farming-simulator-22-mods/implements-and-tools/cultivators-and-harrows/rostelmash-k-12200-cultivator-with-plow-function-v1-0/

Seeder is also a mod yes. Its the Vaederstad Rapid A 6005 / F / M by Maddog Design. Download link is below.

https://fs22.com/farming-simulator-22-mods/implements-and-tools/seeders/vaederstad-rapid-a-600s-f-m-v1-0/

> And there is no logfile for your seeder problem.

Uploading log file now. Here it is with the plow.
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/12795984/log.txt)

> With that plow -> wont fix. If the same happens with a seeder, need log with debug active.

Seeder Log is here. It does also happen with the seeder. I ran the previous log without the seeder debug deactivated, so this log contains both the plow and the seeder.

[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/12796083/log.txt)

Plow Video
Timestamp of the issue starts at 18:13
https://youtu.be/swiAifDXQxg

Seeder Video
Timestamp of the issue starts at 5:43
https://youtu.be/sgJXIroWXJU

**Comment by Tensuko** [comment]
Removed download links.
Dont want to see any other download source then modhub or github.
There are to many pages out there that steal mods and I dont like it that they gain money with the hard work of modders.

Reproduce this with a vanilla tool.

**Comment by BippyMiester** [comment]
Sorry you're correct. I looked at the wrong timestamp. I was looking at the entire video in VLC while I was uploading it, so I must have wrote down the wrong number.

Anyways, I ran the test one more time with the modded plow, however this time i downgraded to 7.3.1.0... issue still occurred so I guess its not a version issue?

So I leased a Cultiplow Platinum 8m. Still on 7.3.1.0, running the test with the vanilla plow. So far so good. You're right in that it could be the plow and the seeder. Will do some further testing over the next few hours and report back if the issue happens again. I'll be recording my gameplay during the entire time and running debug mode for the entire session on the subsoilers/plows and seeders. Ill also put back the most current version at some point and test with that.
